src/assets/styles/_animation.scss

```diff
@@ -71,3 +71,16 @@
 .calcite-animate__in-scale {
   animation-name: in-scale;
 }
+
+@media (prefers-reduced-motion: reduce) {
+  .calcite-animate {
+    opacity: 1;
+    animation: none;
+  }
+}
+
+@media (prefers-reduced-motion: reduce) {
+  :root {
+    --calcite-internal-duration-factor: 0.01s;
```

Comment:
Once [5206](https://github.com/Esri/calcite-components/issues/5206) is installed, we can refactor this to be `0s`